.storybook/config.js

```diff
@@ -0,0 +1,20 @@
+import { configure } from '@storybook/react';
+import { setDefaults } from '@storybook/addon-info';
+
+// addon-info
+setDefaults({
+  header: false,
+  inline: false,
+  source: true,
+  propTablesExclude: [],
+  styles: stylesheet => {
+    stylesheet.link.topRight.zIndex = 99999999;
+    return stylesheet;
+  }
+});
+
+function loadStories() {
+  require('../stories');
+}
+
+configure(loadStories, module);
```

Comment:
Do we need both this file and `stories/index.js`? Both are doing the `loadStories` and `configure`

Reply:
Can this not be inlined instead of creating an (ES5) function?

Reply:
Removed the `index.js` and added it as an inline function.